.gitignore

```diff
@@ -1,7 +1,3 @@
-/.jest
-
-
-
 # See http://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
Can you copy the `.gitignore` file we have in `dev` branch this file shouldn't change since it's not related to NX upgrade.